webpack.config.js

```diff
@@ -17,7 +17,7 @@ const defaultConfig = {
                 use: ['style-loader', 'css-loader'],
             },
             {
-                test: /\.(png|gif)$/,
+                test: /\.(svg|png|gif)$/,
```

Comment:
auf Linie 24 solltest du das svg löschen, sonst funktioniert das (bei mir zumindest) nicht bei den links.

Reply:
Achsoo. Das war die Ursache. Stimmt svg ist doppelt definiert in diesem config